FAQs.md

```diff
@@ -0,0 +1,121 @@
+- Scroll down to library and select either to include or exclude. 
+- Click the plus(+) button and click on the folder you wish to include or exclude then click select.
+
+13. **Do I have to manually scan new tracks into the music player?**
+
+No, Jockey music player automatically scans library for new tracks.
+
+14. **How do I remove an included or excluded folder from the list?**
+On the include/exclude list, click the 3 dots beside the folder and select the remove option.
+
+15. **Can I set my device ringtone and notification sound from Jockey music player?**
+
+No, this feature is not available on Jockey music player. Jockey Music player is built for simplicity and this feature is not a priority.
+
+16. **Can I share songs in my library directly from Jockey music player to other devices?**
+No this feature is not available, although sharing song files can be very useful, it also facilitates piracy which I'd prefer to avoid. 
+
+17. **Can I add and view lyrics of a song on Jockey music player?**
+
+No, lyrics feature isn't available. Jockey Music player is built for simplicity and this feature is quite advanced and also not a priority.
+
+18. **How do I search for a song or playlist?**
+
+To search for a playlist, the search icon can be found at the top bar of the app. Type the song and search.
+
+19. **Can I edit track information on Jockey Music Player?**
+
+No, jockey music player doesn't have a tag editor.
+
+20. **What is sleep timer?**
+
+The sleep timer is a feature that shuts down a device or an app after a preset time has elapsed. On Jockey music player, this feature is intended to let users listen to their favorite music while they sleep. The player shuts down after time has elapsed.
+
+
+21. **Where is the sleep timer feature found and How can I use the sleep timer feature?**
+
+The sleep timer feature option is only visible on the now playing page when the 3 dots on the top right side is clicked. To use the sleep timer, 
+click on the 3 dots on the top right side
+select sleep timer and move the pointer on the roll to set time. The maximum time is 120 minutes(2 hours)
+
+
+22. **Does Jockey music player contain ads?**
+
+No, the app doesn't contain any ads or unnecessary pop-ups
+
+23. **Do I have to pay for any feature on Jockey music player ?**
+
+No, all the features and tools on Jockey music player are available for free.
+
+24. **Where can I download Jockey Music Player?**
+
+The latest release of Jockey can be downloaded from [google playstore](https://play.google.com/store/apps/details?id=com.marverenic.music&hl=en). To view the latest releases and all releases of Jockey music player, click [here](https://github.com/marverenic/Jockey/releases)
+
+
+**If there are other questions you think should be added to this FAQs document, feel free to open an issue or add the question(s) to this document and make a pull request.
```

Comment:
Looks like there's a markdown error here. Let's add the closing `**`'s if you're going for bold text.